DOCS-1247-PGD-6.0-Docs (#6763)
EnterpriseDB/docs: product_docs/docs/pgd/6/reference/tables-views-functions/testingandtuning.mdx

instructs pgd_bench to retry transactions when there's a failover. The `--retry` 
option is automatically enabled when `-m camo` is used.

When using `-m camo` and providing a custom script, the SQL commands in the script must

> be wrapped in SQL transaction commands, i.e. the first SQL command must be `BEGIN`,

Reviewer: ```suggestion
be wrapped in SQL transaction commands. That is, the first SQL command must be `BEGIN`,
```
